packages/forms/src/Commands/MakeFormCommand.php

```diff
@@ -41,7 +41,8 @@ public function handle(): int
             ->map(fn ($segment) => Str::lower(Str::kebab($segment)))
             ->implode('.');
 
-        $model = (string) str($this->argument('model') ?? ($this->option('edit') ?
+        $model = (string) str($this->argument('model') ?? (
+            $this->option('edit') ?
             $this->askRequired('Model (e.g. `Product`)', 'model') :
             $this->ask('(Optional) Model (e.g. `Product`)')
         ))->replace('/', '\\');
```

Comment:
This is quite unreadable to me now. 😅

Reply:
I didn't do anything to this file. Why is it in this?

Reply:
Maybe this was added by github actions.

Reply:
Yeah seems like. Weird it only picked up the changes in this PR. Let's leave it like this for now.